Engine/Setup/BacktestingSetupHandler.cs

```diff
@@ -281,7 +309,7 @@ private TimeSpan GetMaximumRuntime(DateTime start, DateTime finish, int subscrip
         /// <filterpriority>2</filterpriority>
         public void Dispose()
         {
-            // nothing to clean up
+            Compression.ClearZipArchiveCache();
```

Comment:
This looks like some undue coupling. I understand why it's here, but have we tried to run a very long backtest? If we're only clearing this cache at the end of the backtest then presumably we would run out of memory eventually... or I'm misunderstanding how the `Compression.UnzipCached` works